lightbeam/src/backend.rs

```diff
@@ -616,6 +616,7 @@ struct RelocateAccess {
 pub struct TranslatedCodeSection {
     exec_buf: ExecutableBuffer,
     func_starts: Vec<AssemblyOffset>,
+    #[allow(dead_code)]
     relocatable_accesses: Vec<RelocateAccess>,
```

Comment:
I wonder if we should instead remove this field and the corresponding place it is assigned to in `CodeGenSession.into_translated_code_section` rather than leave an unused TODO implementation in place.

Reply:
Although it looks like there's a few other types (`RelocateAddress` and `RelocateAccess`) that would be unused without this field, so I think this is okay.

Reply:
Ok thanks! I noticed this as well. I assumed there was motivation for adding the placeholder so I was reluctant to remove it anyway. I've re-triggered CI.